templates/mocktail/show.html.twig

```diff
@@ -0,0 +1,126 @@
+{% extends 'base.html.twig' %} {% block title %} {{ parent() }} {#
+{{ mocktail.name }} #} {% endblock %} {% block stylesheet %}
+<link rel="stylesheet" href="assets/styles/pages/mocktail_show.css" />
+<link
+    rel="stylesheet"
+    href="//use.fontawesome.com/releases/v5.0.7/css/all.css"
+/>
+{% endblock %} {% block header %}
+<div class="page-heading header-text">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-12">
+                <h1>{{ mocktail.name }}</h1>
+            </div>
+        </div>
+    </div>
+</div>
+{% endblock %} {% block content %}
```

Comment:
please mind the content / body block declared in your base.html.twig. The name is changing from one branch to another and it will create conflicts & bugs. Pick a final name for this block in your base.html.twig then use it in all pages